ipv6/address.go

```diff
@@ -0,0 +1,147 @@
+package ipv6
+
+import (
+	"fmt"
+	"net"
+)
+
+const (
+	// SIZE is the number of bits that an IPv6 address takes
+	SIZE int = 128
+)
+
+// Address represents an IPv6 address
+type Address struct {
+	ui uint128
+}
+
+// AddressFromUint64 returns the IPv6 address from its two 64 bit unsigned representation
+func AddressFromUint64(high uint64, low uint64) Address {
+	return Address{uint128{high, low}}
+}
+
+// AddressFromBytes returns the IPv6 address from bytes
+func AddressFromBytes(s []byte) Address {
+	return Address{Uint128FromBytes(s)}
```

Comment:
I meant to ask this before. Why isn't this implementation *exactly* the same as AddressFromStdIP? In fact, one of them should just type cast to the other and call the same common code.